pkg/api/v1/types.go

```diff
@@ -341,6 +363,9 @@ type MigrationSpec struct {
 type MigrationPhase string
 
 const (
+	// Create Migration has been called but nothing has been done with it
+	MigrationUnknown MigrationPhase = ""
```

Comment:
It means, that so far, no component touched it, if there is no state present ...